### DISCUSSION TITLE
Disable spinner in script

### DISCUSSION THREAD

**Discussion by vrozaksen** [tteck/Proxmox#3433] [Help Needed] [answered]
Hi! I have an issue with the spinner in the interactive Pelican setup. I need a way to disable the spinner for some parts of the setup. It doesn't break the entire process, but the user can't see what they are typing.

![obraz](https://github.com/user-attachments/assets/fb42b174-e812-462a-9c9d-bddc82519950)

**Comment by tteck** [comment, marked as answer]
If that's for a PR, it will not pass.

**Reply by vrozaksen** [reply]
So script need to be fully automatic? Then it will require manual work to get it running.
